google-cloud-testing/spanner-benchwrapper/src/main/java/com/google/cloud/benchwrapper/Main.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.benchwrapper;
+
+import io.grpc.Server;
+import io.grpc.netty.NettyServerBuilder;
+import java.util.Properties;
+
+class Main {
+  public static void main(String[] args) throws Exception {
+    Properties properties = new Properties(System.getProperties());
```

Comment:
Why do you make a copy of this?

Reply:
I got this from the existing code:  
https://github.com/googleapis/google-cloud-java/blob/26415a199764839f4a70640b6a8a7c4b2c953ad9/google-cloud-testing/storage-benchwrapper/src/main/java/com/google/cloud/benchwrapper/Main.java#L25. 

But I agree that this is not necessary. I will update it.